translations/parkour/en.lua

```diff
@@ -27,7 +27,7 @@ translations.en = {
 	help_contribute = "<font size='14'>\n<p align='center'>The parkour management team loves open source code because it <t>helps the community</t>. You can <o>view</o> and <o>modify</o> the source code on <o><u><a href='event:github'>GitHub</a></u></o>.\n\nMaintaining the module is <t>strictly voluntary</t>, so any help regarding <t>code</t>, <t>bug reports</t>, <t>suggestions</t> and <t>creating maps</t> is always <u>welcome and appreciated</u>.\nYou can <vp>report bugs</vp> and <vp>give suggestions</vp> on <o><u><a href='event:discord'>Discord</a></u></o> and/or <o><u><a href='event:github'>GitHub</a></u></o>.\nYou can <vp>submit your maps</vp> in our <o><u><a href='event:map_submission'>Forum Thread</a></u></o>.\n\nMaintaining parkour is not expensive, but it is not free either. We'd love if you could help us by <t>donating any amount</t> <o><u><a href='event:donate'>here</a></u></o>.\n<u>All donations will go towards improving the module.</u></p>",
-	help_changelog = "<font size='13'><p align='center'><o>Version 2.11.0 - 06/02/2022</o></p>\n\n<font size='11'>• <font size='13'>You can now select a hotkey in the options interface for /mort instead of M / Del.",
+	help_changelog = "<font size='13'><p align='center'><o>Version 2.12.0 - 26/07/2023</o></p>\n\n<font size='11'>• <font size='13'>The weekly ranking feature has been reinstated and is now available again.\n<font size='11'>• <font size='13'>Players who achieve top positions in the weekly ranking will now receive their respective badges automatically.\n<font size='11'>• <font size='13'>Several minor bugs have been fixed.",
```

Comment:
I'd recommend changing this date to 31/07 or 01/08; as that is a more realistic release date.
I'll also ask translators to... translate lol